bootstrap-promises/failsafe.cf

```diff
@@ -139,14 +142,15 @@ body copy_from remote(server, path)
         servers    => {
           "${server}"
         };
-        trustkey    => "true";
         source      => "${path}";
         compare     => "digest";
         preserve    => "false";
         verify      => "true";
         purge       => "true";
         copy_backup => "false";
         portnumber  => "5309";
+      trust_key::
```

Comment:
```suggestion
      trust_key.!root_server::
```

Reply:
Why not! (root server is normally never in bootstrap policies but just in case)